conf/nginx.conf

```diff
@@ -0,0 +1,32 @@
+server {
+    server_name localhost;
+    root /var/www/html;
+    index doku.php;
+
+    client_max_body_size 4M;
+    client_body_buffer_size 128k;
+
+    location ~ /(data|conf|bin|inc)/ {
+        deny all;
+    }
+
+    location / { try_files $uri $uri/ @dokuwiki; }
+
+    location @dokuwiki {
+        rewrite ^/_media/(.*) /lib/exe/fetch.php?media=$1 last;
+        rewrite ^/_detail/(.*) /lib/exe/detail.php?media=$1 last;
+        rewrite ^/_export/([^/]+)/(.*) /doku.php?do=export_$1&id=$2 last;
+        rewrite ^/(?!lib/)(.*) /doku.php?id=$1&$args last;
+    }
+
+    location ~ \.php$ {
+        if (!-f $request_filename) { return 404; }
+        include fastcgi_params;
+        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+        fastcgi_param REDIRECT_STATUS 200;
+        fastcgi_pass fpm:9000;
+    }
+
+    error_log /var/log/nginx/error.log;
+    access_log /var/log/nginx/access.log;
```

Comment:
I wonder if these should stay inside the container or if all logs should go to stdout instead

Reply:
The nginx image used in docker-compose.yml already forward those log files to stdout and stderr (see https://github.com/nginxinc/docker-nginx/blob/72ac2226580ee73c170163dbe6e4436373b6ece9/stable/stretch/Dockerfile#L92)
So I think this is ok, isn't it ?